frontend/src/config/ServerConfig.ts

```diff
@@ -169,6 +169,9 @@ export const setServerConfig = (cfg: ServerConfig): void => {
 
   homeCluster = getHomeCluster(serverConfig);
   isMultiCluster = isMC();
+  if (!serverConfig.ambientEnabled) {
+    serverConfig.kialiFeatureFlags.uiDefaults.graph.traffic.ambient = 'none';
```

Comment:
Is this used somewhere else?

Reply:
The value ends up being used when we are setting the traffic options for graph generation requests.  Without this change we will, by default, ask the graph-gen code to include ambient traffic, even if the ambient profile in not enabled.

I considered changing this server-side but decided that it was better to return the real server config from the server, and let clients modify it as they see fit.  We also do some massaging of the health config, so updating the server config does have a precedent.